### ISSUE TITLE
Maintenance: Global, Regional and Country connection

### ISSUE THREAD

**Issue by gclapp1** [energy-data/energydata.info#387]
## Region
Currently, a dataset marked with the regional tag for an area such as Middle East and North Africa (MENA) will not automatically connect to searches for countries in the region i.e Egypt, Tunisia, etc. 

## Global
Global datasets cannot be marked as such and the user must select every region or country to reflect this scope.

**Comment by jodiegardiner** [comment]
1. Trying to make sure i understand the ask.  Currently a dataset only has to have a Region OR a Country. Does this proposed change mean that every datasets must have a Region AND a Country?

**Comment by jodiegardiner** [comment]
2. I added "global" as a region choice.

<img width="944" alt="Screenshot 2023-05-02 at 12 46 25" src="https://user-images.githubusercontent.com/21362845/235657571-fb837db4-2218-448b-83df-892f50989f91.png">

**Comment by jodiegardiner** [comment]
Hmm, one for debate I think!  A dataset could be tagged with MENA but not be anything to do with Egypt could it not?

**Comment by gclapp1** [comment]
Good point- maybe it could auto populate all countries for the region and the user could remove or add as they see fit?
